lib/rules/class-methods-use-this.js

```diff
@@ -42,6 +62,16 @@ module.exports = {
         }
 
         /**
+         * Check if the node is an instance method not excluded by config
+         * @param {ASTNode} node - node to check
+         * @returns {boolean} True if it is an instance method, and included
+         * @private
+         */
+        function isIncludedInstanceMethod(node) {
+            return isInstanceMethod(node) && !Object.prototype.hasOwnProperty.call(exceptMethods, node.key.name);
```

Comment:
That's what I was thinking.

However, node 4 has `Set`, so perhaps I should just use that?